src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/idam/IdamConsumerTestForPostToken.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.hmcts.reform.wataskmonitor.consumer.idam;
+
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ContextConfiguration;
+import uk.gov.hmcts.reform.wataskmonitor.SpringBootContractBaseTest;
+import uk.gov.hmcts.reform.wataskmonitor.clients.IdamWebApi;
+import uk.gov.hmcts.reform.wataskmonitor.domain.idam.Token;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@PactTestFor(providerName = "wa_task_monitor", port = "8892")
+@ContextConfiguration(classes = {IdamConsumerApplication.class})
+public class IdamConsumerTestForPostToken extends SpringBootContractBaseTest {
+
+    @Autowired
+    IdamWebApi idamApi;
```

Comment:
I'm guessing you just copied this but is there any reason why the scope is default here?